postgrest.cabal

```diff
@@ -1,5 +1,5 @@
 name:               postgrest
-version:            9.0.0-a1
+version:            8.0.0.20210921
```

Comment:
I see the version bump was correct - but should this not have triggered a new pre-release right here?

On the release page, I can still only find the `9.0.0-a1` release, but not this `8.0.0.20210921`. Same thing on docker hub.

Should we manually remove the `9.0.0-a1` release everywhere to go full circle on the correction?

Reply:
> but should this not have triggered a new pre-release right here?

Ah, I see - no tag was pushed at the same time...